scripts/import_large_tsv/import_large_data_tables.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
```

Comment:
just pointing out the inconsistency between the file names `import_large_tsv.py` vs `import_large_data_tables.py` - might be clearer to change this to something like `bulk_import_large_tsvs.py`